code/game/objects/items/grenades/spawnergrenade.dm

```diff
@@ -66,3 +66,26 @@
 	name = "clown bug special"
 	spawner_type = /mob/living/simple_animal/cockroach/clownbug
 	deliveryamt = 6
+
+/obj/item/grenade/spawnergrenade/feral_cats
+	name = "feral cat delivery grenade"
+	desc = "This grenade contains 5 dehydrated feral cats in a similar manner to dehydrated monkeys, which, upon detonation, will be rehydrated by a small reservoir of water contained within the grenade. These cats will then attack anything in sight."
+	spawner_type = /mob/living/simple_animal/hostile/feral_cat
+	deliveryamt = 5
+
+/obj/item/grenade/spawnergrenade/feral_cats/prime()			//Own proc for this because the regular one would flash people which was dumb.
```

Comment:
```suggestion
/obj/item/grenade/spawnergrenade/feral_cats/prime()			///Own proc for this because the regular one would flash people which was dumb.
```